server/proxy/volumes.go

```diff
@@ -0,0 +1,85 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package proxy
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/compose-cli/aci"
+	"github.com/docker/compose-cli/api/volumes"
+	volumesv1 "github.com/docker/compose-cli/protos/volumes/v1"
+)
+
+// VolumesCreate creates a volume.
+func (p *proxy) VolumesCreate(ctx context.Context, req *volumesv1.VolumesCreateRequest) (*volumesv1.VolumesCreateResponse, error) {
+	backendClient := Client(ctx)
+	contextType := ""
+	if backendClient != nil {
+		contextType = backendClient.ContextType()
+	}
+
+	if contextType != "aci" {
+		return nil, fmt.Errorf("volume API only available in ACI context")
```

Comment:
You can use not implemented from our errdefs:
```suggestion
		return nil, errors.Wrapf(errdefs.ErrNotImplemented, "volume API only available in ACI context")
```

Reply:
fixed